### ISSUE TITLE
Make it harder to add entries to some reference tables

### ISSUE THREAD

**Issue by robintw** [debrief/pepys-import#613]
## 💬 Brief description of feature

At the moment, users can easily add to reference tables like Nationalities or SensorTypes - but in reality, as we've pre-populated a list of all countries, users shouldn't need to add a new Nationality. In fact, if they're trying to add a nationality it probably means they have just failed to search correctly for the right one (eg. United Kingdom vs Great Britain etc). We should either stop normal users being able to add these, or make it harder (like making it only possible via Pepys Admin, or asking for multiple confirmations).

Discussion is needed as to exactly which tables this should apply to.

### Efficiency of analysis
This will stop duplication of entries in columns like Nationality, which would take time to clean up.

### Effectiveness of analysis
Analysis will be more effective if all data is properly represented in the database, without duplicate entries for things like Nationalities.

## ☑️ Acceptance criteria

[] User is not able to add a Nationality during a file import (or is given a severe warning if they do so)

**Comment by IanMayo** [comment]
Have discussed with users.  Yes, they would like the capability to add a row to reference tables to be removed from Pepys-Admin.  There were aware of a few instances where this had caused a problem.

So, we provide:
```
Ok, please provide platform-type for new Platform:
1) Search for existing platform-type
2) (existing type)
x) (existing type)
.) Cancel import. (Please cancel import & contact an expert Pepys user if you require a new platform-type to be added to Pepys)
```

**Comment by IanMayo** [comment]
Note: this "feature" is actually the removal of a capability. We must ensure it's relatively easy to make these reference table changes in Pepys-Admin.

**Comment by IanMayo** [comment]
Note: Once we've remove "Add" from option 2, that does free up a "slot" in the list of existing types, so we can show one more item.

**Comment by IanMayo** [comment]
Closed in #651
